### PR TITLE
revamp install steps to start nodes in sequence

### PR DESCRIPTION
**Pull request by yussufsh**

### CODE REVIEW

**Review by Prajyot-Parab — DISMISSED** [review]
/lgtm

**Comment by sajauddin** [comment]
".terraform.lock.hcl" Is this file required?

**Comment by yussufsh** [comment]
> ".terraform.lock.hcl" Is this file required?

As per Terraform best practice we should include and update in case of new changes. [Link](https://developer.hashicorp.com/terraform/language/files/dependency-lock#:~:text=You%20should%20include%20this%20file%20in%20your%20version%20control%20repository%20so%20that%20you%20can%20discuss%20potential%20changes%20to%20your%20external%20dependencies%20via%20code%20review%2C%20just%20as%20you%20would%20discuss%20potential%20changes%20to%20your%20configuration%20itself.)
> You should include this file in your version control repository so that you can discuss potential changes to your external dependencies via code review, just as you would discuss potential changes to your configuration itself.

File: modules/4_nodes/nodes.tf [hunk 1]
```diff
@@ -134,7 +141,7 @@ resource "ibm_pi_instance" "master" {
   pi_volume_ids        = local.master.volume_count == 0 ? null : [for ix in range(local.master.volume_count) : ibm_pi_volume.master.*.volume_id[(count.index * local.master.volume_count) + ix]]
 
   # Inject ignition source timeout to force ignition fail when HTTP file is not available for 500s. This will reboot the node and try ignition fetch process again.
```
Comment (sajauddin) [inline modules/4_nodes/nodes.tf:143]: Is this line still holds valid?

Comment (yussufsh) [reply modules/4_nodes/nodes.tf]: fixed.. thanks.

**Review by Prajyot-Parab — APPROVED** [review]
/approve

**Review by sajauddin — APPROVED** [review]

**Pull request merged**